src/main/webapp/deleteProduct.jsp

```diff
@@ -0,0 +1,21 @@
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
```

Comment:
do we need this page for deleting products the same about users?

Reply:
fixed